model/minio/minio_server.rb

```diff
@@ -27,22 +27,18 @@ def private_ipv4_address
     vm.nics.first.private_ipv4.network.to_s
   end
 
-  def name
-    "#{cluster.name}-#{pool.start_index}-#{index}"
-  end
-
   def minio_volumes
     cluster.pools.map do |pool|
       pool.volumes_url
     end.join(" ")
   end
 
-  def connection_string
+  def ip4_url
     "http://#{vm.ephemeral_net4}:9000"
   end
 
   def endpoint
-    dns_zone ? "#{hostname}:9000" : "#{vm.ephemeral_net4}:9000"
+    cluster.dns_zone ? "#{hostname}:9000" : "#{vm.ephemeral_net4}:9000"
```

Comment:
This may not be directly related to this PR, but we might need to modify the hostname format for Minio servers.

For instance, let's say we created a Minio cluster named "my-cluster". The cluster hostname is "my-cluster.minio.com", and the first server hostname is "my-cluster1.minio.com".

What would happen if another customer creates a cluster named "my-cluster1"?

We could consider changing the server hostname format to "my-cluster.1.minio.com" or something similar.

Reply:
For now, we work on minio with the assumption of no customer clusters will be provisioned. We can think about the cluster naming when the time comes.